### PR TITLE
docs: positioning imperative handle `setTarget`

### PR DESCRIPTION
**Pull request by ling1726**
msedge_7ENsOcsfg1

### CODE REVIEW

File: packages/react-positioning/src/types.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export type PopperRefHandle = {
    * Sets the target and updates positioning imperatively.
    * Useful for avoiding double renders with the target option.
    */
-  setTarget: (target: HTMLElement) => void;
+  setTarget: (target: HTMLElement | PopperVirtualElement) => void;
```
Comment (ling1726) [inline packages/react-positioning/src/types.ts:32]: unfortunate bug, the imperative method should also accept virtual elements

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/72b08d56fa19f3b580da0497758af54a6afcf69b'>72b08d56fa19f3b580da0497758af54a6afcf69b</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ContextualMenu-mount_base.html">18298</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ContextualMenu-mount.html">8994</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Avatar-mount_base.html">984</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Avatar-mount.html">955</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/BaseButton-mount_base.html">1031</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/BaseButton-mount.html">971</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Breadcrumb-mount_base.html">2790</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Breadcrumb-mount.html">2876</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ButtonNext-mount_base.html">485</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ButtonNext-mount.html">494</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Checkbox-mount_base.html">1565</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Checkbox-mount.html">1644</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/CheckboxBase-mount_base.html">1368</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/CheckboxBase-mount.html">1363</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ChoiceGroup-mount_base.html">5350</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ChoiceGroup-mount.html">5257</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ComboBox-mount_base.html">1035</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ComboBox-mount.html">1127</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/CommandBar-mount_base.html">11147</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/CommandBar-mount.html">10966</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ContextualMenu-mount_base.html">18298</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ContextualMenu-mount.html">8994</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/DefaultButton-mount_base.html">1194</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/DefaultButton-mount.html">1300</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/DetailsRow-mount_base.html">4027</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/DetailsRow-mount.html">3941</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/DetailsRowFast-mount_base.html">3986</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/DetailsRowFast-mount.html">4043</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/DetailsRowNoStyles-mount_base.html">3757</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/DetailsRowNoStyles-mount.html">3966</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Dialog-mount_base.html">2304</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Dialog-mount.html">2399</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/DocumentCardTitle-mount_base.html">211</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/DocumentCardTitle-mount.html">196</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Dropdown-mount_base.html">3538</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Dropdown-mount.html">3372</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FluentProviderNext-mount_base.html">2014</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FluentProviderNext-mount.html">2029</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FluentProviderWithTheme-mount_base.html">177</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FluentProviderWithTheme-mount.html">175</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">123</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FluentProviderWithTheme-virtual-rerender.html">132</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">199</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">225</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FocusTrapZone-mount_base.html">1895</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FocusTrapZone-mount.html">1966</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FocusZone-mount_base.html">2040</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/FocusZone-mount.html">1946</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/IconButton-mount_base.html">1912</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/IconButton-mount.html">1952</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Label-mount_base.html">393</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Label-mount.html">373</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Layer-mount_base.html">3107</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Layer-mount.html">3251</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Link-mount_base.html">507</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Link-mount.html">510</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/MakeStyles-mount_base.html">1891</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/MakeStyles-mount.html">1781</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/MenuButton-mount_base.html">1565</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/MenuButton-mount.html">1492</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/MessageBar-mount_base.html">2238</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/MessageBar-mount.html">2244</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Nav-mount_base.html">3453</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Nav-mount.html">3601</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/OverflowSet-mount_base.html">1152</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/OverflowSet-mount.html">1256</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Panel-mount_base.html">2383</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Panel-mount.html">2202</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Persona-mount_base.html">927</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Persona-mount.html">936</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Pivot-mount_base.html">1469</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Pivot-mount.html">1587</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/PrimaryButton-mount_base.html">1440</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/PrimaryButton-mount.html">1397</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Rating-mount_base.html">8182</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Rating-mount.html">8493</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/SearchBox-mount_base.html">1513</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/SearchBox-mount.html">1440</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Shimmer-mount_base.html">2723</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Shimmer-mount.html">2630</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Slider-mount_base.html">2177</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Slider-mount.html">2143</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/SpinButton-mount_base.html">5363</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/SpinButton-mount.html">5558</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Spinner-mount_base.html">469</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Spinner-mount.html">508</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/SplitButton-mount_base.html">3243</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/SplitButton-mount.html">3751</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Stack-mount_base.html">589</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Stack-mount.html">633</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/StackWithIntrinsicChildren-mount_base.html">2552</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/StackWithIntrinsicChildren-mount.html">2394</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/StackWithTextChildren-mount_base.html">5685</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/StackWithTextChildren-mount.html">5652</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/SwatchColorPicker-mount_base.html">12507</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/SwatchColorPicker-mount.html">12136</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/TagPicker-mount_base.html">2950</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/TagPicker-mount.html">2682</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/TeachingBubble-mount_base.html">13990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/TeachingBubble-mount.html">14143</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Text-mount_base.html">464</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Text-mount.html">486</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/TextField-mount_base.html">1481</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/TextField-mount.html">1491</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ThemeProvider-mount_base.html">1341</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ThemeProvider-mount.html">1264</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ThemeProvider-virtual-rerender_base.html">692</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ThemeProvider-virtual-rerender.html">689</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2084</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1969</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Toggle-mount_base.html">855</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/Toggle-mount.html">902</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/buttonNative-mount_base.html">166</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21875/perf-test/buttonNative-mount.html">165</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by micahgodbolt — APPROVED** [review]

File: packages/react-components/src/Concepts/Positioning/PositioningSetTarget.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        height: 0,
+        top: y,
+        right: x,
+        bottom: y,
+        left: x,
+        x,
+        y,
+      });
+    };
+    const virtualElement = {
```
Comment (layershifter) [inline packages/react-components/src/Concepts/Positioning/PositioningSetTarget.stories.tsx:22]: There should be a type for this. Can we re-export it and use it?

Comment (ling1726) [reply packages/react-components/src/Concepts/Positioning/PositioningSetTarget.stories.tsx:22]: Yeah I can re-export from react - positioning, we still have the open issue for reexpoting these from the suite

Comment (ling1726) [reply packages/react-components/src/Concepts/Positioning/PositioningSetTarget.stories.tsx]: actually it's already exported.... I just forgot to use it 🤦

**Review by bsunderhus — APPROVED** [review]
LGTM

**Review by andrefcdias — APPROVED** [review]

**Pull request merged**